website_timecheck/models/res_config_settings.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    homepage_url = fields.Char(
+        string="Homepage", related="website_id.homepage_url", readonly=False
```

Comment:
What is the expected notation here? (Add help as appropriate.) Would adding `widget="url"` suit?

Reply:
Added Help and widget="url" to the xml.